consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/QbftBlockHeaderValidationRulesetFactory.java

```diff
@@ -58,7 +57,8 @@ public static BlockHeaderValidator.Builder blockHeaderValidator(
             new GasLimitRangeAndDeltaValidationRule(
                 DEFAULT_MIN_GAS_LIMIT, DEFAULT_MAX_GAS_LIMIT, baseFeeMarket))
         .addRule(new TimestampBoundedByFutureParameter(1))
-        .addRule(new TimestampMoreRecentThanParent(secondsBetweenBlocks))
+        //.addRule(new TimestampMoreRecentThanParent(secondsBetweenBlocks))
+        // removed because of conflict with emptyblockperiodseconds
```

Comment:
Can you explain the reason for this? This looks like you've removed the requirement for new blocks to have a timestamp > than old blocks, but I would expect that to continue to be true whether it's empty or not? If it should be removed, it just be removed entirely, not commented out. People can use git history to see the removal.

Reply:
I have to revisit this, IIRC the issue is with the secondsBetweenBlocks that is always set to the blockperiodseconds, so the rule failed whenever the emptyblockperiodseconds > blockperiodseconds. But this was on my initial version based on a older version of Besu, I'll uncomment it and check if the issue's still there...

Reply:
I have renstated the TimestampMoreRecentThanParent header rule, I did a bunch of tests with several diferent scenariosd and all went well. I did had a negative result but I think it was because I was running it under debugger. The rule totally ignore the emptyblockperiodseconds but since either the emptyblockperiodseconds is allways > than blockperiodseconds or being lower is ignoired in favour of blockperiodseconds it seems to alwayd evaluate correctly.